org.godotengine.Godot.yaml

```diff
@@ -3,9 +3,19 @@ runtime: org.freedesktop.Platform
 runtime-version: '19.08'
 default-branch: stable
 sdk: org.freedesktop.Sdk
+# Use writable-sdk to install MSBuild for Mono
+writable-sdk: true
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono5
 command: godot
 
 build-options:
+  append-path: /usr/lib/sdk/mono5/bin
+  append-ld-library-path: /usr/lib/sdk/mono5/lib
+  build-args:
+    - --share=ipc
+    - --socket=x11
```

Comment:
No idea if OpenGL can work but try running under `xvfb-run`?

Reply:
I tried, but the `xvfb-run` is not in the runtime.

Reply:
You can bundle it only for the build. `cleanup: ["*"]` will delete it afterwards.

Reply:
Yes, I will try today after sleeping haha.

Reply:
Using `Xvfb`, I got that error:

```
Godot Engine v3.2.1.stable.mono.flathub - https://godotengine.org
ERROR: initialize: X11 Display is not available
   At: platform/x11/os_x11.cpp:132.
```

Reply:
You can generate the intermediate binaries with platform=server, then you don't need X11 to generate te glue. Just make sure you generate the final binaries with platform=x11 again.

Reply:
I sent a new PR but it didn't trigger a build. :confused:

Reply:
The error now: dotnet/msbuild#2423 / godotengine/godot#40204.